src/main/java/stream/flarebot/flarebot/commands/general/PollCommand.java

```diff
@@ -1,19 +1,353 @@
+                            try {
+                                color = ColorEnum.getColorByName(args[3]).get();
+                            } catch (NoSuchElementException ex) {
+                                MessageUtils.sendErrorMessage("That is not a valid color input!", channel);
+                                return;
+                            }
+                        }
+                        channel.sendMessage(new EmbedBuilder().setColor(color)
+                                .setDescription("Changed the color of the poll to `" + GeneralUtils.colourFormat(color) + "`")
+                                .build())
+                                .queue();
+                        poll.setColor(color);
+                        return;
+                    }
+
+                }
+            } else if (args[0].equalsIgnoreCase("options")) {
+                if (args.length >= 3) {
+                    Poll poll = getPollById(args[1], guild.getPolls(), channel);
+                    if (poll == null) {
+                        return;
+                    }
+
+                    if (args[2].equalsIgnoreCase("view") && args.length == 4) {
+                        int optionId;
+                        PollOption option;
+
+                        try {
+                            optionId = Integer.parseInt(args[3]) - 1;
+                            option = poll.getPollOptions().get(optionId);
+                        } catch (NumberFormatException | IndexOutOfBoundsException e) {
+                            MessageUtils.sendErrorMessage("Please provide a valid poll option ID!\n" +
+                                    "Use the poll option list command to see the IDs!", channel);
+                            return;
+                        }
+
+                        channel.sendMessage(MessageUtils.getEmbed(sender)
+                                .setDescription("Information for option ID: " + args[1])
+                                .setColor(Color.CYAN)
+                                .addField("Option", GeneralUtils.truncate(1024, option.getOption(), false), false)
+                                .addField("Votes", String.valueOf(option.getVotes()), false)
+                                .build()).queue();
+                        return;
+                    } else if (args[2].equalsIgnoreCase("add") && args.length >= 4) {
+                        String option = MessageUtils.getMessage(args, 3);
+                        List<PollOption> options = poll.getPollOptions();
+                        if (options.size() >= 20) {
+                            MessageUtils.sendErrorMessage("You can only have up to 20 options on one poll!", channel);
+                            return;
+                        } else if (option.length() > 1024) {
+                            MessageUtils.sendErrorMessage("The option you entered is too long! It has to be 1024 characters or shorter!", channel);
```

Comment:
I do remember, I truncate this in embeds to make sure it doesn't go over the limit, the only one that displays it in full is the option view